src/gui/__init__airline.py

```diff
@@ -0,0 +1,279 @@
+import customtkinter as ctk
+from data.airline_repository import AirlineRepository  # Importing AirlineRepository
+
+class App:
+    def __init__(self, airline_repo: AirlineRepository):
```

Comment:
Unfortunately this will not work. I understand this is for testing but we can't add this to the codebase. The reason is that we already have an App in `__init__.py` that gets execute and it's really hard to merge this into that. We will need to have separate components for each section

Reply:
Understand. Saw the changes you have made in another branch.